src/screens/TokenDetail.js

```diff
@@ -286,6 +336,47 @@ class TokenDetail extends React.Component {
       );
     }
 
+    const renderIcons = () => {
+      if (this.state.unknown) {
+        return (
+          <div>
+            <i className="fa fa-warning text-warning ml-3" title="Token is not registered"></i>
+            <i className="fa fa-check pointer ml-3" title="Register token" onClick={this.registerClicked}></i>
+          </div>
+        );
+      } else {
+        return (
+          <div>
+            <i className="fa fa-trash pointer ml-3" title="Unregister token" onClick={this.unregisterClicked}></i>
+          </div>
+        );
+      }
+    }
+
+    const renderWalletInfo = () => {
+      if (this.state.unknown) {
+        return null;
+      } else {
+        return (
+          <div className="token-detail-wallet-info">
+            <h4 className="mt-3 mb-2"><strong>Wallet data</strong></h4>
```

Comment:
With this new info on the token details screen, we might want to add a link to a page where we explain what each of them mean (which we don't have yet).

We also probably need to add more text to explain that the top part is about the token in general and the bottom (wallet data) is what you have on your wallet.

Reply:
I don't know if I like this page with the explanation (but maybe we should do it). Maybe just adding more text to this page could solve the doubts and (if we do) some short tutorial videos explaining it.

I tried to make this page as clean as possible, we could try to add also a title to the top part (something like 'General token data' don't know) and an (?) icon with explanation of each one of them.

Really don't know what's best

Reply:
I think adding the amount, mint and melt authorities right after the token name at the top and the other could be named 'About this token', after this first part (so inverting the current order).

But, as there's a lot of information on it, I still think we should link to a page where everything is explained (maybe a video like you mentioned).

Reply:
All screens have changed a lot. Can you check again, please?